packages/core-utils/src/utils/assertions/assert-event.ts

```diff
@@ -21,11 +21,11 @@ export default async function assertEvent(
           : ''
       })`;
 
+      seenEvents.push(text);
+
```

Comment:
If there is a partial match, seenEvents would never be populated, and the console would show something like:

Expected event "ABC()" but found "" (nothing).